.github/workflows/solidity.yml

```diff
@@ -43,7 +43,7 @@ jobs:
 
       - name: Set cache key
         id: set-cache-key
-        run: echo "cache-key=ci-solidity-${{ hashFiles('**/*.sol') }}" >> "$GITHUB_OUTPUT"
+        run: echo "cache-key=ci-solidity-${{ hashFiles('contracts') }}" >> "$GITHUB_OUTPUT"
```

Comment:
Since the ABIs are created uniquely based on sol files, I think this would create more cache keys than necessary.